### PR TITLE
Add fedora 31;rearrange package manager links

### LINKED ISSUE #16281: Fedora 31
There no page for fedora 31.

---
#### Сведения о документе

⚠ *Не вносите правки в этот раздел. Это необходимо для связывания страницы сайта docs.microsoft.com с вопросом на GitHub.*

* ID: 3e656d98-d9c6-3feb-c663-96894c813213
* Version Independent ID: 39aa858c-592b-d57f-eb78-759d5add5f35
* Content: [Install .NET Core on Fedora 30 - package manager - .NET Core](https://docs.microsoft.com/ru-ru/dotnet/core/install/linux-package-manager-fedora30#feedback)
* Content Source: [docs/core/install/linux-package-manager-fedora30.md](https://github.com/dotnet/docs/blob/master/docs/core/install/linux-package-manager-fedora30.md)
* Product: **dotnet-core**
* GitHub Login: @Thraka
* Microsoft Alias: **adegeo**

### PR DESCRIPTION
**Pull request by Thraka**
## Summary

- Added Fedora 31 package manager page
- Added Fedory 31 links
- Rearranged package manager into
  1. Highest version to lowest
  2. Ubuntu first then others alpha

Fixes #16281

On hold until new year when people are back.

### CODE REVIEW

File: docs/core/install/linux-package-manager-fedora31.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- register the product repository
+- Install required dependencies
+
+This only needs to be done once per machine.
+
+Open a terminal and run the following commands.
+
+```bash
+sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
+sudo wget -q -O /etc/yum.repos.d/microsoft-prod.repo https://packages.microsoft.com/config/fedora/31/prod.repo
```
Comment (Youssef1313) [inline docs/core/install/linux-package-manager-fedora31.md]: The `https://packages.microsoft.com/config/fedora/31/prod.repo` link gives 404 not found.

File: docs/core/install/linux-package-manager-fedora31.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+[!INCLUDE [package-manager-switcher](./includes/package-manager-switcher.md)]
+
+This article describes how to use a package manager to install .NET Core on Fedora 31. If you're installing the runtime, we suggest you install the [ASP.NET Core runtime](#install-the-aspnet-core-runtime), as it includes both .NET Core and ASP.NET Core runtimes.
+
+## Register Microsoft key and feed
+
+Before installing .NET, you'll need to:
+
+- Register the Microsoft key
+- register the product repository
```
Comment (Youssef1313) [inline docs/core/install/linux-package-manager-fedora31.md]: Capitalize the "R" to be consistent with the other list items. And added a period.

```suggestion
- Register the product repository.
```

File: docs/core/install/linux-package-manager-fedora31.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+[!INCLUDE [package-manager-switcher](./includes/package-manager-switcher.md)]
+
+This article describes how to use a package manager to install .NET Core on Fedora 31. If you're installing the runtime, we suggest you install the [ASP.NET Core runtime](#install-the-aspnet-core-runtime), as it includes both .NET Core and ASP.NET Core runtimes.
+
+## Register Microsoft key and feed
+
+Before installing .NET, you'll need to:
+
+- Register the Microsoft key
```
Comment (Youssef1313) [inline docs/core/install/linux-package-manager-fedora31.md]:
```suggestion
- Register the Microsoft key.
```

File: docs/core/install/linux-package-manager-fedora31.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+- register the product repository
+- Install required dependencies
```
Comment (Youssef1313) [inline docs/core/install/linux-package-manager-fedora31.md]:
```suggestion
- Install required dependencies.
```

Comment (Thraka) [reply docs/core/install/linux-package-manager-fedora31.md]: For single sentence lists, you don't need a . according to the MS Style Guide (I just learned that.)

File: docs/core/install/linux-package-manager-fedora31.md (hunk 1 above)
Comment (Thraka) [reply docs/core/install/linux-package-manager-fedora31.md]: Yes, this is in WIP until I find out if/when it's coming.

File: docs/core/install/linux-package-manager-fedora31.md (hunk 4 above)
Comment (Youssef1313) [reply docs/core/install/linux-package-manager-fedora31.md]: @Thraka, Hmm, I see the example in the style-guide does the opposite.

https://docs.microsoft.com/en-us/style-guide/scannable-content/lists#bulleted-lists

Comment (mairaw) [reply docs/core/install/linux-package-manager-fedora31.md]: @Youssef1313's link is what I use as a reference for lists. Can you point to where on the style guide says we're not supposed to add periods @Thraka? The exception would be if there are 3 words or less (https://docs.microsoft.com/en-us/style-guide/punctuation/periods).

File: docs/core/install/linux-package-manager-fedora31.md (hunk 1 above)
Comment (Youssef1313) [reply docs/core/install/linux-package-manager-fedora31.md]: @Thraka, It's available now. The link is no longer 404.

File: docs/core/install/linux-package-manager-fedora31.md (hunk 4 above)
Comment (Thraka) [reply docs/core/install/linux-package-manager-fedora31.md]: https://styleguides.azurewebsites.net/Styleguide/Read?id=2696&topicid=28112

But perhaps I was misinterpreting it.

Comment (mairaw) [reply docs/core/install/linux-package-manager-fedora31.md]: Yeah, usually more specific style guides can override the Microsoft style guide. I think the key here is to determine whether these are complete sentences or not.

Comment (Thraka) [reply docs/core/install/linux-package-manager-fedora31.md]: I just merged a PR that fixes all of the other files too.

File: docs/core/install/linux-package-manager-fedora31.md (hunk 1 above)
Comment (Thraka) [reply docs/core/install/linux-package-manager-fedora31.md]: The actual packages will be published around a week from now, then we'll merge this PR.

**Comment by Thraka** [comment]
@tdykstra This is ready. Been waiting on the packages to be published, which happened last night.

**Review by tdykstra — APPROVED** [review]
LGTM

**Pull request merged**